.flayignore

```diff
@@ -5,3 +5,5 @@ lib/gooddata/lcm/actions/collect_data_product.rb
 # TMA-652
 lib/gooddata/lcm/actions/synchronize_user_filters.rb
 spec/integration/date_dim_switch_spec.rb
+spec/unit/actions/collect_segments_spec.rb
+spec/unit/actions/collect_segment_clients_spec.rb
```

Comment:
You should really never ever add files to .flayignore

Reply:
The flay errors I get from those files are not reasonable. Ignoring the whole file is the only way to ignore errors in flay. Therefore, I propose to allow adding specs to `.flayignore` until there is a better way to avoid unreasonable flay errors.

cc @kubamahnert

Reply:
Agreed wiith @Hahihula and @kubamahnert to solve this in https://jira.intgdc.com/browse/TMA-984